modules/flowable-engine/src/test/java/org/flowable/engine/test/api/task/TaskQueryTest.java

```diff
@@ -1268,6 +1268,20 @@ public void testQueryByUnassignedOr() {
         assertThat(query.count()).isEqualTo(11);
         assertThat(query.list()).hasSize(11);
     }
+    
+    @Test
+    public void testQueryByAssigned() {
+        TaskQuery query = taskService.createTaskQuery().taskAssigned();
+        assertThat(query.count()).isEqualTo(1);
+        assertThat(query.list()).hasSize(1);
+    }
+
+    @Test
+    public void testQueryByAssignedOr() {
+        TaskQuery query = taskService.createTaskQuery().or().taskId("invalid").taskAssigned();
+        assertThat(query.count()).isEqualTo(0);
```

Comment:
It seems like the `or` is not working correctly. This should have a count of `1` since there is 1 assigned task.

Reply:
Fixed above issue.